system/service/audit_service.py

```diff
@@ -49,12 +49,87 @@ def create_audit(self, document_uid: str, auditor_username: str) -> Optional[Dic
         """
         user = self.user_repo.find_user_by_username(auditor_username)
         if user:
+            audit_status = AuditStatus(
+                name=auditor_username,
+                audit_status_value=3 #Possible values: "approved(1)", "rejected(2)", "pending(3)"
+            )
+            new_audit_status = self.audit_repo.create_audit_status(audit_status)
+            current_app.logger.info(f"New audits status record {new_audit_status}")
             audit = Audit(
                 uid=str(uuid4()),
                 document_id=document_uid,
                 creator_id=user.id,
-                audit_status_id=1
+                audit_status_id=audit_status.id
             )
             new_audit = self.audit_repo.create_audit(audit)
+            current_app.logger.info(f"New audits record {new_audit}")
             return {"auditUid": new_audit.uid}
-        return None
\ No newline at end of file
+        return None
+
+    def get_audit_result(self, document_uid: str) -> Optional[Dict]:
```

Comment:
TODO: add docstring

Reply:
Solved in [1fc3d62](https://github.com/NYCU-CloudNativeTeam/DocumentSystem/pull/1/commits/1fc3d6226434c0883ec375afc6e7fa8d45a12d3b)